src/relayclient/RelayClient.ts

```diff
@@ -370,6 +391,12 @@ export class RelayClient {
 
     return forwarderAddress
   }
+
+  verifyInitialized (): void {
```

Comment:
I don't think it is important to make this state observable at all. It is an internal detail, and R.C. should be able to cope with it if you don't enforce `await rc.init()` being done first

Reply:
To be read as 'please don't' 😀

Reply:
Please see where I used it.
e.g. addAccount/newAccount: these are non-async (and I don't want to make them so), so I can't do init().
also send
But I also don't want them to crash on strange "no member X of undefined" or something like that.

Reply:
But these methods do not require initialization, do they? I object to making `verifyInitialized` part of RelayClient API. Vehemently.